enterprise/server/tasksize/tasksize.go

```diff
@@ -23,6 +23,12 @@ const (
 	DefaultCPUEstimate      = int64(600)
 	DefaultFreeDiskEstimate = int64(100 * 1e6) // 100 MB
 
+	// Additional resources needed depending on task characteristics
+
+	FirecrackerAdditionalMemEstimateBytes = int64(150 * 1e6) // 150 MB
+	DockerAdditionalMemEstimateBytes      = int64(800 * 1e6) // 800 MB
```

Comment:
Can you clarify (either via comment or more explicit variable name) that this is the additional memory for running Docker-in-firecracker than anything to do with the the Docker container type.

Reply:
Done